### DISCUSSION TITLE
System Wide Configuration

### DISCUSSION THREAD

**Discussion by c0rydoras** [qtile/qtile#4483] [Ideas]
Support system wide config 
1. `$XDG_CONFIG_HOME`
2. `~/.config`
3. system wide place e.g. `/etc/xdg/`

**Comment by m-col** [comment]
If this is helpful and you're happy to work on this, then I'd support having this behaviour :)

The i3wm implementation of this is may be of use: https://github.com/i3/i3/blob/next/libi3/get_config_path.c

**Reply by c0rydoras** [reply]
Here would be a first draft: https://github.com/qtile/qtile/pull/4490
